### DISCUSSION TITLE
Just a dumb question from a beginner: clink.symbols.zip What for?

### DISCUSSION THREAD

**Discussion by n0valis** [chrisant996/clink#608] [General]
I saw the zip file containing the pdb symbol files. What are they good for?

**Comment by chrisant996** [comment]
Google it.

**Reply by n0valis** [reply]
Hey chrisant996, 
thanks for your patience and expertise. I really appreciate people like you. I suppose you're eagerly awaiting my reply. Here you go: I did

nice gf btw
